joomla/joomla-cms
# [js message system] Correct js messages class

#### Summary of Changes

Simple PR to correct the colors and titles of the js messages.
##### Before

![image](https://cloud.githubusercontent.com/assets/9630530/15306281/1bd7cafa-1bc1-11e6-8b4b-b52d533e3c7f.png)
##### After

![image](https://cloud.githubusercontent.com/assets/9630530/15306280/196281a2-1bc1-11e6-9ceb-3d30d7e4c44f.png)
#### Testing Instructions
- Go to isis/protostar index.php and add this code

``` php
// JS Messages
JText::script('MESSAGE');
JText::script('ERROR');
JText::script('NOTICE');
JText::script('WARNING');

JFactory::getDocument()->addScriptDeclaration("
jQuery(document).ready(function() {
    var messages = {
        'message': ['JS Success Message 1', 'JS Success Message 2'],
        'error': ['JS Error Message 1', 'JS Error Message 2'],
        'notice': ['JS Notice Message 1', 'JS Notice Message 2'],
        'warning': ['JS Warning Message 1', 'JS Warning Message 2']
    };
    Joomla.renderMessages(messages);
});");
```
- Now run any isis/protostar page. You'll see the "Before" scenario
- Apply patch
- Repeat step 2 (clear browser cache). You will see the "After" scenario.
